plane/source/docs/tuning-cruise.rst

```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
+
+.. note:: The :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` parameter should be adjusted to the average throttle value used at cruise speed. It optimizes the bias point for the speed control loops and is used in case of airspeed failure.
+
+While cruising, the artificial horizon in the OSD or GCS may show an average positive or negative pitch above the level indicator (ie fuselage/autopilot level). This means that the "trim level" pitch or AOA is different than what was set during the accelerometer calibration step. This can be trimmed out so that non altitude controlled modes fly at the same speed and throttle. Do this by adjusting the pitch trim, adding the desired degrees nose up or down using :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. This also optimizes the speed control loop bias point and will allow non altitude controlled modes to fly level at the same throttle and speed. 
+
+
+Without an Airspeed Sensor
+--------------------------
+
+Without an airspeed sensor, both the pitch trim and the :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` parameter would need to be changed appropriately for the desired mid-stick cruise speed. 
+
+.. tip:: Often small light planes or gliders need 2 or 3 degrees of pitch trim to fly at their optimum cruising speed/throttle rather than at the fuselage/autopilot level pitch. This can be done at setup by:
+
+- **(Preferred)** Add the desired degrees nose up(usually) or down to :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. 
+- Position vehicle with a few degrees nose up or down during the first, Level step of accelerometer calibration to match the cruising attitude.
+- Position vehicle with a few degrees nose up and use the  ``Calibrate Level`` button on the Mission Planner page.T his adjusts the AHRS_TRIM parameters. AHRS_TRIM parameters can only change the difference between the autopilot's plane and "level" by 10 degrees maximum. If more is needed, (e.g. the autopilot is mounted slightly downward), then you can use :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to alter the AOA manually.
+
+.. tip:: You can examine ATT.Pitch in the logs when at cruise speed in FBWB or CRUISE to determine the average pitch trim required in these modes. Appropriately adjusting :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to lower this to zero when flying level in these modes.
+
+.. note:: when using :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to adjust trim, it will be reflected in the ATT.Pitch log message, and also in the OSD and GCS horizon displays (ie level horizon pitch = :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` + the calibrated level pitch) so that the display will be level when flying "in trim" even though the plane's pitch is different than the autopilots calibrated pitch. If it is desired to actually reflect the autopilots pitch, use :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` bits 8 and/or 9  to show this in OSD and/or GCS artificial horizon.
```

Comment:
Did those options make it into a stable release?

Reply:
no,and already in Upcoming...will remove for this